ThirdParty/cppzmq/zmq.hpp

```diff
@@ -297,6 +297,11 @@ inline int poll(std::vector<zmq_pollitem_t> &items, long timeout_ = -1)
 }
 #endif
 
+inline bool has_message(const zmq_pollitem_t &item)
+{
+    return ((static_cast<uint32_t>(item.revents) & ZMQ_POLLIN)!=0U);
+}
+
```

Comment:
Is this something in the upstream version of cppzmq?

Reply:
It cleared up a clang-tidy warning and simplified some code.  So it isn't in the upstream, but there are a few other functions we have like that as well.   I wanted to have it be inline and it needed to have access to zmq.h which isn't true for a lot of the other headers so this seemed like the most logical place at the moment.